app/helpers/user_menu_helper.rb

```diff
@@ -0,0 +1,17 @@
+module UserMenuHelper
+  def profile_user_menu_link
+    link_to t(:profile), user_path
+  end
+
+  def messages_user_menu_link
+    link_to t(:messages), messages_user_path
+  end
+
+  def discussions_user_menu_link
+    link_to t(:discussions), discussions_user_path
+  end
+
+  def should_show_discussions?
+    Organization.current.forum_enabled?
+  end
```

Comment:
This is a change I proposed: instead of showing the discussions tab (now discussions link) only if the user has made one or subscribed to one, it now works like messages - you can see it, but it's empty. I added a message as well (see `:discussions_will_be_here`), and we can change that text if it's not very good too.

This is now a tad inconsistent with the behavior on the menu bar (see https://github.com/mumuki/mumuki-laboratory/blob/master/app/helpers/discussions_helper.rb#L14-L16) which only shows `Mis consultas` if one has already been made. But we can change that too anyway.

Reply:
Awesome! We could open for a _quick_ debate whether to show messages and discussions only if there are messages/discussions, but intuitively I would say that your decision is correct